tools/cherry-picker/entrypoint.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh -l
+/cherry-picker/node_modules/.bin/ts-node --project /cherry-picker/tsconfig.json /cherry-picker/src/main.ts
```

Comment:
Can you use the npm script for that? `npm run cherry-picker`?

Reply:
I could, but this would be two commands then. 
``` 
cd /cherry-picker
npm run ript for that? npm run --project /cherry-picker/tsconfig.json /cherry-picker/src/main.ts
```
Is that better in any way?

Reply:
if you change a path once you would only have to adapt on one place ;) – no code duplication

Reply:
but nevertheless, the code is working and this is my personal opinion so for me is everything fine!